FirebaseSegmentation.podspec

```diff
@@ -0,0 +1,41 @@
+Pod::Spec.new do |s|
+  s.name             = 'FirebaseSegmentation'
+  s.version          = '0.1.0'
+  s.summary          = 'Firebase Segmentation SDK'
+  s.description      = <<-DESC
+Firebase Segmentation enables you to associate your custom application instance ID with Firebase for user segmentation.
+                       DESC
+
+  s.homepage         = 'https://firebase.google.com'
+  s.license          = { :type => 'Apache', :file => 'LICENSE' }
+  s.authors          = 'Google, Inc.'
+  s.source           = {
+    :git => 'https://github.com/firebase/firebase-ios-sdk.git',
+    :tag => 'Segmentation-' + s.version.to_s
+  }
+
+  s.ios.deployment_target = '8.0'
+  
+  s.cocoapods_version = '>= 1.4.0'
+  s.static_framework = true
+  s.prefix_header_file = false
+
+  s.source_files = 'Segmentation/Sources/**/*'
+  s.public_header_files = 'Segmentation/Sources/Public/*.h'
+
+  s.dependency 'FirebaseCore', '~> 6.0'
+  s.dependency 'FirebaseInstanceID', '~> 4.1.1'
```

Comment:
Should only depend on minor versions. Change to 4.1

Reply:
Done.